lib/beaker-answers/answers.rb

```diff
@@ -64,19 +63,19 @@ class Answers
         'orchestrator_database_user'     => DEFAULT_ANSWERS[:q_orchestrator_database_user],
         'orchestrator_database_password' => DEFAULT_ANSWERS[:q_orchestrator_database_password],
         'use_application_services'       => true,
-      }
+      },
```

Comment:
Hmm, is this a style that we want to go with, the trailing comma for the last key-pair of a hash?

It's not my favorite thing, but, I'm not really against it either.

Reply:
It makes it a lot easier to add new key-pairs. Not strongly in favor, but it does seem convenient.